### ISSUE TITLE
Some cmx files are not installed

### ISSUE THREAD

**Issue by copy** [c-cube/ocaml-containers#88]
I'm seeing warning 58 in a project:

```
File "_none_", line 1:
Warning 58: no cmx file was found in path for module CCString, and its interface was not compiled with -opaque
```

From a quick look at `.opam/4.04.0+flambda/lib/containers/`, it seems that the cmx produces by cppo sources are not installed.

**Comment by c-cube** [comment]
Yes, it's a known issue due to oasis, which does not install .cmx for files that do not exist at configure time: `CCString` (and `CCFun`) are generated by cppo during compilation. In other words, I will wait for oasis to fix that ;-)

**Comment by c-cube** [comment]
Should be fixed by moving to jbuilder (see #165)

**Comment by c-cube** [comment]
Actually fixed by the removal of cppo.
